inst/schema/TableMetadata.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "definitions": {
+    "table_preset": {
+      "type": "object",
+      "properties": {
+        "label": { "type": "string" },
+        "column": { "type": "string" },
+        "row": { "type": "string" },
+        "selected_filter_options": {
+          "type": "object",
+          "patternProperties": {
+            "^.*$": {
+              "type": "array",
+              "items": { "type": "string" }
+            }
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [ "label", "column", "row" ]
+    }
+  },
+  "properties" : {
+    "presets" : {
```

Comment:
We don't the option to also include any default filter values outside the presets? Those will just be whatever the defaults are for the choropleth?

Reply:
the defaults in the frontend are actually just the first values in the filters array :laughing: so we should be alright